samcli/local/apigw/local_apigw_service.py

```diff
@@ -230,6 +235,17 @@ def _parse_lambda_output(lambda_output, binary_types, flask_request):
 
         return status_code, headers, body
 
+    @staticmethod
+    def _invalid_apig_response(output):
+        allowable = [
+            "statusCode",
+            "body",
+            "headers",
+            "isBase64Encoded"
+        ]
+        invalid_keys = set(output) - set(allowable)
```

Comment:
nit: Is `output` dict? If so explicitly grab the keys using `output.keys()`.

Reply:
Updated, and tests still passing locally.